Http/Controllers/Backend/ContentsController.php

```diff
@@ -38,6 +38,7 @@ public function getAssets(Request $request, $content_slug)
         $data['non_repeatable_fields'] = Content::getNonRepeatableFields();
 
         $data['content_type'] = $request->content_type;
+
```

Comment:
@we-vineet-k001 don't commit such files with no changes

Reply:
sorry sir, And I will remember this next time